src/main/accumulMod.F90

```diff
@@ -252,7 +264,12 @@ subroutine init_accum_field (name, units, desc, &
     else
        accum(nf)%type2d = ' '
     end if
-    
+    if (present(scale_by_thickness)) then
+       accum(nf)%scale_by_thickness = scale_by_thickness
+    else
+       accum(nf)%scale_by_thickness = ' '
```

Comment:
- [ ] Isn't scale_by_thickness a logical? So this should be set to .false. rather than ' '?